.azuredevops/tag-builds/AMDMIGraphX.yml

```diff
@@ -1,6 +1,6 @@
 variables:
 - group: common
-- template: /.azuredevops/variables-global.yml@pipelines_repo
```

Comment:
We removed all pipelines_repo usage but still define it. Is it refactored somewhere?

Reply:
Where do you see it defined? It is implied that the current repo is the pipelines_repo (i.e., ROCm/ROCm).

Reply:
I'm a little confused about this, pipelines_repo seems like a variable defined by us. The first time it appears is in tag-builds/*.yml under resources section (please correct me if I'm wrong). So I thought this was the definition and later when we do @pipelines_repo, we specifying repo as ROCm/ROCm.